test/SourceKit/Macros/macro_basic.swift

```diff
@@ -0,0 +1,50 @@
+macro stringify<T>(_ value: T) -> (T, String) = MacroDefinition.StringifyMacro
+
+func testStringify(a: Int, b: Int) {
+  _ = #stringify(a + b)
+}
+
+// REQUIRES=shell
+
+// RUN: %empty-directory(%t)
+
+//##-- Prepare the macro plugin.
+// RUN: %target-build-swift -I %swift-host-lib-dir -L %swift-host-lib-dir -emit-library -o %t/%target-library-name(MacroDefinition) -module-name=MacroDefinition %S/../../Macros/Inputs/syntax_macro_definitions.swift -g -no-toolchain-stdlib-rpath
+
+// RUN: COMPILER_ARGS=( \
+// RUN:   -enable-experimental-feature Macros \
+// RUN:   -load-plugin-library %t/%target-library-name(MacroDefinition) \
+// RUN:   -I %swift-host-lib-dir \
+// RUN:   -module-name MacroUser \
+// RUN:   %s \
+// RUN: )
+
+//##-- cursor-info at '#' position.
```

Comment:
Should being on the `#` mean we don't give back the macro decl?

Reply:
Right, currently `#` is not a part of the "reference", so we don't give back the macro decl for `#` position.